plugin/pkg/admission/limitranger/admission.go

```diff
@@ -451,6 +487,28 @@ func PodLimitFunc(limitRange *api.LimitRange, pod *api.Pod) error {
 			}
 			podRequests := sum(containerRequests)
 			podLimits := sum(containerLimits)
+			for j := range pod.Spec.InitContainers {
+				container := &pod.Spec.InitContainers[j]
+				// take max(sum_containers, any_init_container)
+				for k, v := range container.Resources.Requests {
+					if v2, ok := podRequests[k]; ok {
+						if v.Value() > v2.Value() {
```

Comment:
this is a bad means to do comparison because it loses precision.

can you look at using https://github.com/kubernetes/kubernetes/blob/master/pkg/quota/resources.go#L50

it will let you know what keys to swap.